src/Groups/action.jl

```diff
@@ -267,3 +267,43 @@ function on_indeterminates(f::Nemo.MPolyElem, s::PermGroupElem)
 end
 
 ^(f::Nemo.MPolyElem, p::PermGroupElem) = on_indeterminates(f, p)
+
+
+#############################################################################
+##
+##  natural stabilizers in permutation groups
+
+function stabilizer(G::PermGroup, pnt::T) where T <: Oscar.IntegerUnion
+    return Oscar._as_subgroup(G, GAP.Globals.Stabilizer(G.X, GAP.GapObj(pnt), GAP.Globals.OnPoints))
+end
+
+function stabilizer(G::PermGroup, pnt::Vector{T}) where T <: Oscar.IntegerUnion
+    return Oscar._as_subgroup(G, GAP.Globals.Stabilizer(G.X, GAP.GapObj(pnt), GAP.Globals.OnTuples))
+end
+
+function stabilizer(G::PermGroup, pnt::Set{T}) where T <: Oscar.IntegerUnion
+    pnt = [x for x in pnt]
+    sort!(pnt)
+    return Oscar._as_subgroup(G, GAP.Globals.Stabilizer(G.X, GAP.GapObj(pnt), GAP.Globals.OnSets))
+end
+
+
+#############################################################################
+##
+##  natural stabilizers in matrix groups
+
+function stabilizer(G::MatrixGroup{ET,MT}, pnt::AbstractAlgebra.Generic.FreeModuleElem{ET}) where {ET,MT}
+    img = GAP.GapObj(G.ring_iso.(pnt.v))[1]
+    return Oscar._as_subgroup(G, GAP.Globals.Stabilizer(G.X, img, GAP.Globals.OnRight))
```

Comment:
Alternatively one could use the optional `gens, acts` arguments with the GAP generators and then the Oscar generators, then no ring isomorphism is needed, and one could even compute orbits this way.

Reply:
Yes. Apparently there are no problems on the GAP side in the case of `stabilizer`. This means that we can get a generic method, see the version which I have just pushed,